code/game/objects/items/clown_items.dm

```diff
@@ -58,6 +58,15 @@
 	cleanspeed = 28 //janitor gets this
 	uses = 300
 
+//Yogs start - Adds infinite soap
+/obj/item/soap/infinite

```

Comment:
Do this in ``yogstation/code/game/objects/items/clown_items.dm`` and just set the uses to something really high. The ``if(uses != -1)`` checks are really stupid.

Reply:
are you sure because like soapstone, crayon and spraycan have uses = -1 for infinite amounts, and it's not really infinite if it's just a high number

Reply:
Admin rcd is infinite but not really,it’s a very high number and I couldn’t get to empty it

Reply:
Use the INFINITY label, it’s high

Reply:
yeah, I will, but still not true infinity :disappointed:

Reply:
You cannot drain the soap within a human lifetime